src/libraries/System.Management/src/System/Management/ManagementScope.cs

```diff
@@ -1181,7 +1181,7 @@ internal int GetObject_(string strObjectPath, int lFlags, IWbemContext pCtx, ref
             //It is assumed that caller always passes ppCallResult as IntPtr.Zero.
             //If it changes let this call go through wminet_utils.dll. Check implementation of CreateInstanceEnum_ for more information.
             int status = (int)tag_WBEMSTATUS.WBEM_E_FAILED;
-            if (!object.ReferenceEquals(ppCallResult, IntPtr.Zero))
```

Comment:
This ReferenceEquals will _never_ be true, which means the body of this `if` block will have always executed, and now with your change it won't necessarily.  That's concerning in a library for which we have close to zero test coverage.

Reply:
Yep, agreed. I noticed that already. What do you recommend as next steps? I wanted to take _some_ action to improve things based on the static analysis, but didn't want to increase the scope of the change too much. In some of the areas I don't have enough expertise/context to make the right judgement call. Remove the code? Try to add tests to fill the gap? Leave things as is?

Reply:
I would leave the code functionally as is, which means either deleting the dead code (the always true condition) or reverting the change entirely. If you can also add tests, great.

>  I wanted to take some action to improve things based on the static analysis

Good goal. Sometimes the right action is to ignore it / do nothing. You have a lot more context than the tool does.